tools/samples-build

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+#
+# This script builds the sample applications.
+
+set -e
+
+pushd samples/cpp
```

Comment:
Can you add different option to run? like build-sample all/cpp/go/android, I think we will run them separately, our pipelines are dedicated to different platforms. Android Pipeline couldn’t build iOS or mac sample

Reply:
I have overlooked this.  I added argument, for different platforms, linux/ios/android/all.  I thought this should be added in one new build, but I see now that this is not feasible.  It could be added to 3 *new* CI builds (run in parallel), or to the existing linux/osx/android builds as an extra step.